README.md

```diff
@@ -204,228 +215,251 @@ It is possible to configure the `macvlan` driver to bridge your container to the
+_Note:_ When you run as a non-root user,
+you will not be able to bind to lower ports by default.
+(This would not necessary if you are using the default ports.)
+If you must do this, also pass the 
+`--sysctl net.ipv4.ip_unprivileged_port_start=0`
+option on the `docker run...` to bind to whatever port you wish.
 
-Default `1024M`
+~~## Unifi-in-Docker on Windows~~
 
-### External MongoDB environment variables
+~~Unifi uses the Mongo database to store its data.
+Mongo uses the fsync() system call on its data files.
+Because of how Docker for Windows works, you can't bind mount `/unifi/db/data` on a Docker for Windows container.~~
 
-These variables are used to implement support for an [external MongoDB server](https://community.ubnt.com/t5/UniFi-Wireless/External-MongoDB-Server/td-p/1305297) and must all be set in order for this feature to work. Once all are set then the configuration file value for `db.mongo.local` will automatically be set to `false`.
+~~The article [MongoDB on Windows in Minutes with Docker](https://blog.jeremylikness.com/blog/2018-12-27_mongodb-on-windows-in-minutes-with-docker/) also provides guidance for setting up a Docker Volume to work with Mongo database.~~
 
-### `DB_URI`
+~~As a simpler alternative,
+use the `docker-compose` command as described
+in the next section since it uses a pre-built
+Mongo container that addresses the problem.~~
 
-Maps to `db.mongo.uri`.
+## Running with separate Mongo container
 
-### `STATDB_URI`
+_[ Should we remove this section?
+The simple all-in-one container above works just fine
+for people who "just want it to work" ]_
 
-Maps to `statdb.mongo.uri`.
+The `docker-compose.yml` file in this repository provides a
+single command that orchestrates all the actions required
+to bring up Mongo and Unifi Controller in separate containers,
+using named volumes for important directories.
+(No need to worry about the fsync() problem described above.)
 
-### `DB_NAME`
+Simply clone this repo or copy the `docker-compose.yml` file
+to your host computer's local disk and run:
 
-Maps to `unifi.db.name`.
+```bash
+cd <directory with docker-compose.yml>
+docker-compose up -d 
+```
 
+**Setting Options:**
 
-## Expose:
+* The `docker-compose.yml` file contains the options
+passed to Unifi Controller when it starts.
+Edit the `docker-compose.yml` file setting its values according to the 
+[Options on the command line.](#options-on-the-command-line)
+* _Optional:_ Add additional `-e <any-environment-variables-you-want>` to the `docker-compose up` line
 
-This container exposes the following ports. A minimal Unifi Controller requires the first three.
+To change options to Unifi Controller::
 
-* 8080/tcp - Device command/control
+```bash
+cd <directory with docker-compose.yml>
+docker-compose down # this stops Unifi Controller and MongoDB
+# ... edit the options in the docker-compose.yml file ...
+docker-compose up ... # to resume operation
+```
 
-* 8443/tcp - Web interface + API
+## Beta Users
```

Comment:
It might be worth it to remove the beta section for now, beta is somewhat broken unless run as root right now and needs some work to fix that.

Reply:
OK